lib/datura/to_es/vra_to_es/fields.rb

```diff
@@ -109,17 +107,14 @@ def medium
   end
 
   def person
-    # TODO will need some examples of how this will work
-    # and put in the xpaths above, also for attributes, etc
-    # should contain name, id, and role
-    eles = @xml.xpath(@xpaths["person"])
-    eles.map do |p|
+    eles = get_elements(@xpaths["person"]).map do |p|
       {
-        "id" => "",
+        "id" => p["id"],
         "name" => Datura::Helpers.normalize_space(p.text),
         "role" => Datura::Helpers.normalize_space(p["role"])
```

Comment:
There should not be a difference, no. This example is just using nokogiri to access the inner text and `@role` but they should probably be switched to `get_text` since I'm typically trying to use that instead to be friendlier to non-devs!

Reply:
Okay it turns out there's a slight difference, which is that it will return "" instead of nil, but I don't think that's a problem